src/Metadata/Property/Factory/ExtractorPropertyMetadataFactory.php

```diff
@@ -48,9 +48,11 @@ public function create(string $resourceClass, string $property, array $options =
             }
         }
 
+        $isInterface = interface_exists($resourceClass);
+
         if (
-            !property_exists($resourceClass, $property) ||
-            !$propertyMetadata = $this->extractor->getResources()[$resourceClass]['properties'][$property] ?? false
+            !property_exists($resourceClass, $property) && !$isInterface ||
+            empty($propertyMetadata = $this->extractor->getResources()[$resourceClass]['properties'][$property] ?? [])
```

Comment:
null === (... ?? null)

Reply:
:+1: you right. (can't remember why I did it like that, probably a mistake after testing things)